docs/source/examples/workspaces/aws/aws.key

```diff
@@ -0,0 +1,3 @@
+[key]
```

Comment:
Remove this file please, and add `**/*.key` to .gitignore to avoid future incedents.